src/CppAst/CppModelBuilder.cs

```diff
@@ -40,9 +42,12 @@ public CXChildVisitResult VisitTranslationUnit(CXCursor cursor, CXCursor parent,
 
             _rootContainerContext.Container = _rootCompilation;
 
-            _isEntryVisitSystem = cursor.Location.IsInSystemHeader;
+            
             if (cursor.Location.IsInSystemHeader)
             {
+                if (!ParseSystemIncludes) return CXChildVisitResult.CXChildVisit_Continue;
+
+                _isEntryVisitSystem = true;
```

Comment:
Shouldn't `_isEntryVisitSystem ` be set to false in case `IsInSystemHeader` is false?

Reply:
👍 with that said where is `_isEntryVisitSystem` actually used ?

Reply:
haha, yeah, for some reason, it seems that I had a plan to do something with that... but obviously, this plan was aborted! 😅  You can remove it as part of this PR then!

Reply:
👍 Done